yb-voyager/src/migassessment/common.go

```diff
@@ -39,10 +39,13 @@ type SizingAssessmentReport struct {
 	MemoryPerInstance               float64
 	OptimalSelectConnectionsPerNode int64
 	OptimalInsertConnectionsPerNode int64
-	MigrationTimeTakenInMin         float64
-	ParallelVoyagerThreadsSharded   int64
-	ParallelVoyagerThreadsColocated int64
-	FailureReasoning                string
+	ImportTimeTakenInMin            float64
+	ParallelVoyagerThreads          float64
```

Comment:
Shall we call it `ParallelVoyagerJobs` i.e. jobs instead of threads (even in the report) ? 
Threads would be an implementation detail. Furthermore, since it's golang, it's not even "threads" technically!

Reply:
done. Changed it to ```ParallelVoyagerJobs```